sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -4,205 +4,350 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
-import { InteractiveBrowserCredential, BrowserLoginStyle } from "@azure/identity";
-import { KeyClient, KeyVaultKey } from "@azure/keyvault-keys";
-
-const getRandomKeyName = () => `key${Math.random()}`.replace(/\./g, "");
+import {
+  InteractiveBrowserCredential,
```

Comment:
With the default options, I'm running into this error.
![image](https://user-images.githubusercontent.com/10452642/107592449-d61cbd80-6bc1-11eb-9963-275e99fff3e7.png)

Reply:
Seems to be working fine with the artifact from #13263